src/core/lib/iomgr/tcp_server_custom.cc

```diff
@@ -80,9 +81,19 @@ static grpc_error* tcp_server_create(grpc_closure* shutdown_complete,
                                      const grpc_channel_args* args,
                                      grpc_tcp_server** server) {
   grpc_tcp_server* s = (grpc_tcp_server*)gpr_malloc(sizeof(grpc_tcp_server));
+  // Let the implementation decide if so_reuseport should be enabled or not.
+  s->so_reuseport = true;
```

Comment:
if I understand correctly, this is enable reuseport by default. Are you sure we want to do this?

Reply:
This aligns with the behavior for `tcp_server_posix` which will check if SO_REUSEPORT can be set, if so, it will enable it by default.